src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/IRazorNotificationHandler.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts
 {
-    internal interface IRazorNotificationHandler<TRequestType> : INotificationHandler<TRequestType, RazorRequestContext>
+    internal interface IRazorNotificationHandler<RequestType> : INotificationHandler<RequestType, RazorRequestContext>
```

Comment:
Don't make me tap the sign

Reply:
Yes, please _always_ prefix generic type parameters with `T`. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/generic-type-parameters#type-parameter-naming-guidelines

Reply:
Wish that were an analyzer so the IDE would alert me.